static/src/javascripts/projects/commercial/modules/dfp/lazy-load.ts

```diff
@@ -36,26 +35,36 @@ const onIntersect = (
 		(advert) => !advertIds.includes(advert.id),
 	);
 };
-/**
- * return a random number between 20 (inclusive) and 420 (non-inclusive)
- * @returns number
- */
-const getRandomLazyLoadMargin = (): number => {
-	const min = 20;
-	const max = 420;
-	return Math.floor(Math.random() * (max - min)) + min;
-};
+
+const lazyLoadMargins = {
+	'variant-1': 20,
+	'variant-2': 70,
+	'variant-3': 120,
+	'variant-4': 170,
+	'variant-5': 220,
+	'variant-6': 270,
+	'variant-7': 320,
+	'variant-8': 370,
+} as const;
+
+type LazyLoadMarginTestVariant = keyof typeof lazyLoadMargins;
 
 const getObserver = once(() => {
-	const inLazyLoadMarginTestVariant = isInVariantSynchronous(
-		commercialLazyLoadMargin,
-		'variant',
-	);
+	const lazyLoadMarginTestVariants = Object.keys(lazyLoadMargins).filter(
+		(variantName) => {
+			return isInVariantSynchronous(
+				commercialLazyLoadMargin,
+				variantName,
+			);
+		},
+	) as LazyLoadMarginTestVariant[];
+	const lazyLoadMarginTestVariant: LazyLoadMarginTestVariant | null =
+		lazyLoadMarginTestVariants.length > 0
+			? lazyLoadMarginTestVariants[0]
+			: null;
```

Comment:
You may be able to simplify this using `find` instead of `filter`. Haven't tested this but something like...

```suggestion
	const lazyLoadMarginTestVariant = Object.keys(lazyLoadMargins).find(
		(variantName) => {
			return isInVariantSynchronous(
				commercialLazyLoadMargin,
				variantName,
			);
		},
	);
```

Reply:
Good call! That's much simpler 😌